tests/conftest.py

```
@@ -0,0 +1,9 @@
+import sys
+from pathlib import Path
+
+
+def pytest_configure(config):
```

Ruff (0.14.14)

[warning] Unused function argument: `config`

(ARG001)